source/clear-linux/get-started/virtual-machine-install/virtualbox-cl-installer.rst

```diff
@@ -1,258 +1,384 @@
+#. Validate the integrity of the downloaded image by checking the file hash
+   and signatures. Refer to the document on :ref:`validate-signatures` for
    detailed steps.
 
-#. Decompress the downloaded image. Uncompressed image size is ~ **5GB**.
+#. Decompress the downloaded image.
 
-   - On Windows you can use `7zip`_ to extract the file by right-clicking the 
+   - On Windows you can use `7zip`_ to extract the file by right-clicking the
      file to *Extract Here* (in the same directory)
 
-       .. image:: ./figures/vbox/vbox-extract-cl-ISO.png
-          :alt: 7zip extract here command
+   .. figure:: figures/vbox/virtualbox-cl-installer-01.png
+      :scale: 100%
+      :alt: 7zip extract here command
+
+      Figure 1: 7zip extract here command
 
    - On Linux :
 
      .. code-block:: bash
 
-         xz -d clear-<VERSION>-installer.iso.xz
-
-
-
-
-#. The originally downloaded compressed archive file 
-   (:file:`clear-<VERSION>-installer.iso.xz`) can now be deleted.
+        xz -d clear-<VERSION>-live-server.iso.xz
 
+#. Delete the originally downloaded compressed file.
 
 Create a new |VB| virtual machine
 *********************************
 
-A new :abbr:`VM (Virtual Machine)` needs to be created in |VBM| for |CL| to be installed onto. 
-
-General instructions for creating a virtual machine and details about using 
-different settings are available on the 
+A new :abbr:`VM (Virtual Machine)` needs to be created in |VBM| where |CL|
+will be installed. General instructions for creating a virtual machine and
+details about using different settings are available on the
 `VirtualBox manual section on Creating a VM`_.
 
-
 #. Launch the |VBM| from your host system.
 
+#. Click the *New* button to create a new VM.
 
-#. Click the *New* button to create a new VM. 
-
-   .. image:: ./figures/vbox/vbox-new-vm.png
+   .. figure:: figures/vbox/virtualbox-cl-installer-02.png
+      :scale: 100%
       :alt: Create a new VM in VirtualBox
 
+      Figure 2: Create a new VM in VirtualBox
 
-#. A *Create Virtual Machine* window will appear. 
+#. A *Create Virtual Machine* window will appear.
    Select the following settings:
-   
+
+   - Name: Enter a name (e.g. ClearLinuxOS).
    - Type: **Linux**
    - Version: **Linux 2.6 / 3.x / 4.x (64-bit)**
-   - Memory size: **1024 MB** (this can be adjusted appropriately)
-   - Hard disk: **Create a virtual hard disk now**
 
-   .. image:: ./figures/vbox/vbox-create-vm-new-disk.png
-      :alt: Create a new VM in VirtualBox with a new disk
+#. Enter a memory size.
 
+   - Memory size default: **2048 MB** (Adjust appropriately.)
 
-#. Click the *Create* button.
+   .. note::
 
-#. A *Create Virtual Hard Disk* window will appear. 
-   Select the following settings:
+      Later, if you want to change the amonunt of RAM allocated, power down your VM. Return to :file:`Settings > System` and change
+      :guilabel:`Base Memory` to the desired size.
+
+#. In :guilabel:`Hard disk`, select
```

Comment:
I've redone all screenshots several times. You're welcome to follow up with another PR if you like.

Reply:
@puneetse @tpleavitt, You're both asking for opposite use cases. I suggest that we push this as soon as questions on technical accuracy are resolved. If you want to improve screenshots, use an expert mode, etc., I suggest you create a follow up PR. As it stands, we need to support our user base and respond to the basic need of assuring that this document "works", not that it's perfect.

Reply:
I disagree, I rather not create more work later and have a delta on quality. If it is really important to merge now, a follow up PR would look like a rebase from the previous version if I'm being honest. 

The source of confusion is probably started at this mass change when the rest of the documents were updated to point to the GUI installer: https://github.com/clearlinux/clear-linux-documentation/commit/5ff59f975d77512dc13534fc037f703955ac85aa#diff-e833b80e9e1044868cb31dfb06533c8aR187 . The rest of the virtualbox instructions were using/implying the TUI installer but the link to the installer instructions started pointing to live desktop.

This simplest way to update this doc would be to continue using the same flow, update to using the likekind installer (live-server), and update the screenshots according (I make this statement by successfully following the exact steps in the current document and just changing these 2 things). As it sits now, the verbiage is split between the two installers as @tpleavitt points out.

Reply:
Who do we think is the primary reader?  A person using live-server or one using live-desktop and the GUI.  I believe it will be the latter hence my request to focus on the live-desktop and point out the differences as appropriate for the live-server.  Let's not get hung up on this - this document needs to get updated soon.  If it is easier to keep the live-server then so be it and my preference is known.  Just remove the conflicting mix of both.

Reply:
Done.